CMakeLists.txt

```diff
@@ -5,26 +5,6 @@ endif()
 
 project(s25client)
 
-if(NOT MSVC)
-    #Check for ccache
-    if(APPLE)
-        find_program(CCACHE_PROGRAM NAMES ccache-lipo ccache)
```

Comment:
we should add "buildcache" to the list (i.e usable for VS builds)

Reply:
Got to be done in libutil now :)
However I'm not sure there is a benefit. First time I hear about buildcache, so who uses it? Also it seems it (max) disable parallel compiler invocation for MSVC: https://github.com/mbitsnbites/buildcache/blob/master/doc/usage.md#using-with-visual-studio--msbuild

Reply:
Well yes, but its quiet nice to use (i.e I do use it at work ;-))